tools/cli-generator/generate.go

```diff
@@ -237,7 +237,8 @@ func cleanupFile(filePath string) error {
 	if err != nil {
 		return err
 	}
-	return os.WriteFile(filePath, r, os.ModePerm)
+	const filePerm = 0600
```

Comment:
Do we need this change as part of this PR?

Reply:
I think you also changed this here: #3101

Reply:
Yes I did, removed it from this PR now